core/src/banking_stage/committer.rs

```diff
@@ -112,15 +104,21 @@ impl Committer {
                 &commit_results,
                 Some(&self.replay_vote_sender),
             );
+
+            let committed_transactions = commit_results
+                .iter()
+                .zip(batch.sanitized_transactions())
```

Comment:
I believe using `commit_results` here instead of `processing_results` is valid based on how the below function translates a `TransactionProcessingResult` to a `TransactionCommitResult`:
https://github.com/anza-xyz/agave/blob/6eaa62a517eb660e1fbf41e5afc9e40713bace95/runtime/src/bank.rs#L3835-L3892

Reply:
Agree that using `commit_results` is harmless, it also more consistent with `blockstore_processor` code path.